core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -19,6 +19,98 @@
 var waitFor = require('../protractor_utils/waitFor.js');
 
 var ExplorationEditorTranslationTab = function() {
+  var dismissWelcomeModalButton = element(
+    by.css('.protractor-test-dismiss-welcome-modal'));
+  var translationWelcomeModal = element(
+    by.css('.protractor-test-welcome-modal'));
+
+  /*
```

Comment:
Can we remove this comments or add this in more detail?

Reply:
line 28-31.

Reply:
Yeah we can remove this